README.md

```diff
@@ -1,6 +1,102 @@
 # Agriboard
+
+Agriboard is an administrative dashboard for the agriculture software stack. It is used to manage users and view advanced analytics.
+
+## Dependencies
+
+### NodeJS
+
+To get started, the first thing you're going to need NodeJS (version `8.9.0` or higher). You can download this [here](https://nodejs.org/en/)
+
+### Yarn
+
+You will also require Yarn to manage the javascript dependencies in the application. You can view installation instructions for your OS [here](https://yarnpkg.com/lang/en/docs/install/)
+
+To install dependencies simply run `yarn`
+
+**Important:** Please use yarn to manage your dependencies. **You should never run any of the following commands:**
+
+- `npm install`
+- `npm install <package>`
+- `npm install --dev <package>`
+- `npm remove <package>`
+
+Instead use:
+
+- `yarn`
+- `yarn add <package>`
+- `yarn add --dev <package>`
+- `yarn remove <package>`
+
+## Scripts
+
+### Development
+
+Getting started developing is easy, simply run:
+
+`npm start`
+
+Hot reloading is enabled, so there is no need to restart the server or reload the page when you make changes. They will automatically propogate to your browser, while maintaining the previous state of your application.

```

Comment:
Not sure if it is a good idea but we could specify in the readme to open up localhost:<port> in their browser. It probably says which port in the console, but it could be helpful here as well for beginners :)

Reply:
Should we have this in the `REAME`? It will display this in the console when they start the app